Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -831,3 +851,23 @@ extension ShortcutItem {
     }
   }
 }
+
+private func liveStreamData(fromNavigation nav: Navigation)
+  -> SignalProducer<(Project, Project.LiveStream, LiveStreamEvent, RefTag?), NoError> {
+
+    guard case let .project(projectParam, .liveStream(eventId), refTag) = nav else { return .empty }
+
+    return SignalProducer.zip(
```

Comment:
this is a fun use of `zip`! we need to call out to the main kickstarter api and the live stream api to fetch some data, so might as well do it concurrently!